paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -73,15 +74,31 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
             configuration = paymentSheetConfiguration,
         )
 
-        reportLoadResult(elementsSessionResult)
-
         elementsSessionResult.mapCatching { elementsSession ->
+            val isGooglePayReady = async {
```

Comment:
I think we're missing some test coverage for changes in this file. Can we add a test for the analytics being fired?